tests/data/scad/stl/stl-import-limits.stl

```diff
@@ -0,0 +1,86 @@
+solid OpenSCAD_Model
+  facet normal -1 0 0
+    outer loop
+      vertex 0 0 0
+      vertex 0 0 1.23456788
+      vertex 0 1.23456788 1.23456788
+    endloop
+  endfacet
+  facet normal 0 0 -1.47171605e-07
```

Comment:
Why is there a leading zero on all the exponents?  Is `double-conversion` actually outputting that for you, and this test is being run succeessfully?  I don't think it should.

e.g.
```
echo(1.19208998e-07); // 1.19209e-7
```

Reply:
The above comment was a small nitpick for the purpose of just minimizing ASCII filesize... but actually thinking about it some more, this appears to be an error that the normal is not normalized to a unit vector.

Reply:
Ah yes I forgot to copy the output back to the input (same file is in expectations and input folder). This was from the earlier %.9g formats